composer.json

```diff
@@ -97,7 +94,7 @@
         "spatie/laravel-cookie-consent": "^2.4",
         "spatie/laravel-permission": "^3.0",
         "symfony/translation": "^5.0",
-        "vatsim/sso": "dev-develop#7579e9f492d84f04bfd30df41f0073b3d7a01527",
+        "vatsim/sso": "3.*",
```

Comment:
Can we be any more specific, maybe to a minor version at least? I know there shouldn't be any breaking changes, but that makes the dangerous assumption of SEMVER being followed to the spec.

Reply:
21655fec